src/main/java/eu/dissco/backend/controller/DigitalMediaObjectController.java

```diff
@@ -125,14 +126,16 @@ public ResponseEntity<JsonApiListResponseWrapper> getMasJobRecordForMedia(
   @PostMapping(value = "/{prefix}/{suffix}/mas", produces = MediaType.APPLICATION_JSON_VALUE)
   public ResponseEntity<JsonApiListResponseWrapper> scheduleMassForDigitalMediaObject(
       @PathVariable("prefix") String prefix, @PathVariable("suffix") String suffix,
+      @RequestParam Optional<Boolean> batching,
```

Comment:
I think you can set a Default value for this:
`@RequestParam(defaultValue = false) boolean batching`

Reply:
I think sonar suggested doing it this way instead (or maybe it was the IDE...?) Can't trigger the recommendation again, so I'll set a defaultValue instead.